packages/shared/ReactFeatureFlags.js

```diff
@@ -39,6 +39,9 @@ export const warnAboutLegacyContextAPI = false;
 // Gather advanced timing metrics for Profiler subtrees.
 export const enableProfilerTimer = __PROFILE__;
 
+// If the DevTools hook is present, inject into it.
+export const supportDevToolsIfPresent = false;
```

Comment:
Shouldn't this be true? I think this caused devtools code to get DCE'd in open source builds.